### ISSUE TITLE
Old step is not available in 'show.bs-stepper' event details

### ISSUE THREAD

**Issue by nelson6e65** [Johann-S/bs-stepper#59]
Hi. Amazing package.

The 'show' event is cancellable, but only is provided the step that will be 'shown' via `event.detail.indexStep`. I would like to know the `old` and `next` step in this event.

~I would like to know how to check the current step (before 'shown') in the 'show' event?~

I am able to use the global `stepperInstance._currentIndex` as the `old` value, but... should be nice to have it as event data.

**Comment by Johann-S** [comment]
Hi @nelson6e65,

Thanks for your feedback, yes you can use `_currentIndex` or you do: `event.detail.indexStep - 1` to know the previous step.

I'll keep your issue for a next major release of bs-stepper, because if we do that it'll introduce a breaking change

**Comment by nelson6e65** [comment]
Hmm... That is cool for `.next()`, but what about `previous()` and `.to(2)` methods?

I think introduce `oldValue` property and an alias `newValue` in addition to current `indexStep`, it should not break anything.

```js
{
  oldStep,
  newStep, // Alias for indexStep
  indexStep, // current property
}
```

**Comment by Johann-S** [comment]
I agree for `.to` it's impossible to know which was the old step.

Feel free to make a PR or I'll do it when I find the time 👍

**Comment by Johann-S** [comment]
Fixed by: https://github.com/Johann-S/bs-stepper/commit/16b146aa5f94c481bfd572e4207f180b63b02d47

**Comment by nelson6e65** [comment]
Nice! Thanks. 🎉

**Comment by Johann-S** [comment]
you're welcome 😉 it's already shipped in v1.7.0
